Library/ViewModels/RewardCardContainerViewModelTests.swift

```diff
@@ -168,6 +182,7 @@ final class RewardCardContainerViewModelTests: TestCase {
       "No longer available",
       "No longer available",
       "No longer available",
+      "No longer available",
```

Comment:
These tests are a little hard to read - I should probably refactor them sometime, the intention was to try to cover a bunch of different project states and reward configurations but it would probably be easier to understand if we just broke them up.

Nevertheless, this is the test that asserts the change we made - we added the `availableLimitedUnavailableTimebasedReward` and in the arrays of true/false and strings in this test we see that the second last value that we added has the expected result.